concordia/static/css/base.css

```diff
@@ -1234,14 +1234,15 @@ div.related-links {
     #contribute-container {
         border-style: none;
     }
-    #contribute-container #viewer-column {
-        height: 300px;
+    #contribute-container #viewer-column .print-transcription-image {
+        height: 700px;
```

Comment:
Most of our pages probably want to be larger than 700px. I'm wondering whether this should be something like a calc expression to be 100% minus the header height so you can get as much image as will fit on one page.

Reply:
Ok, both `100%` and `100vh` height translates inconsistently on Chrome, Firefox and IE. I think what we can do is place the image at the very bottom of the page, and let it scale to the full width.